### PR TITLE
Decouple add base url support

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Related to Avaiga/taipy-designer#143

### CODE REVIEW

File: frontend/taipy-gui/base/src/app.ts [hunk 1]
```diff
@@ -252,6 +253,10 @@ export class TaipyApp {
     getWsStatus() {
         return this._ackList;
     }
+
+    getBaseUrl() {
+        return getBase();
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/base/src/app.ts:258]: Why not... rename `getBase()`  to `getBaseUrl()`?

File: frontend/taipy-gui/base/src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const getBase = () => {
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/base/src/utils.ts:1]: Same thing here. Use the same terminology for the same concept...

**Review by FabienLelaquais — DISMISSED** [review]
Waiting to see Designer...

File: frontend/taipy-gui/base/src/app.ts (hunk 1 above)
Comment (dinhlongviolin1) [reply frontend/taipy-gui/base/src/app.ts:258]: I dont want it to conflict with the original getBaseUrl from taipy-gui hence the shortened version. I can still change it back though

**Review by FabienLelaquais — APPROVED** [review]
👍

**Pull request merged**